test/test_distributions.py

```diff
@@ -514,6 +520,22 @@ def test_cauchy(self):
         loc.grad.zero_()
         scale.grad.zero_()
 
+    def test_half_cauchy(self):
+        loc = Variable(torch.zeros(5, 5), requires_grad=True)
+        scale = Variable(torch.ones(5, 5), requires_grad=True)
+        loc_1d = Variable(torch.zeros(1), requires_grad=True)
+        scale_1d = Variable(torch.ones(1), requires_grad=True)
+        self.assertEqual(HalfCauchy(loc, scale).sample().size(), (5, 5))
+        self.assertEqual(HalfCauchy(loc, scale).sample_n(7).size(), (7, 5, 5))
+        self.assertEqual(HalfCauchy(loc_1d, scale_1d).sample().size(), (1,))
+        self.assertEqual(HalfCauchy(loc_1d, scale_1d).sample_n(1).size(), (1, 1))
+        self.assertEqual(HalfCauchy(0.0, 1.0).sample_n(1).size(), (1,))
+
+        set_rng_seed(1)
+        self._gradcheck_log_prob(Uniform, (loc, scale))
```

Comment:
Replace `Uniform` with `HalfCauchy`. If you feel like cleaning up, also fix this error in `test_cauchy()`.

Reply:
done